### PR TITLE
[FIX] web: products snippet: fix layout issues

### PR DESCRIPTION
**Pull request by xlu-odoo**
Before this commit, some templates for the products snippet in website had some layout issues. 
This commit fixes them.

task-3060991

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/website_sale/data/product_snippet_template_data.xml [hunk 1]
```diff
@@ -320,14 +322,14 @@
                         <img class="img img-fluid position-absolute w-100 h-100 o_img_product_cover" loading="lazy" t-att-src="data['image_512']" t-att-alt="record.display_name"/>
                     </a>
                     <div class="o_carousel_product_card_body d-flex flex-column justify-content-between h-100 bg-black-50 p-3 position-relative">
-                        <div class="mb-3">
+                        <div class="mb-1">
```
Comment (Brieuc-brd) [inline addons/website_sale/data/product_snippet_template_data.xml:325]: Decreasing this margin creates a spacing inconsistency when the rating stars are displayed.
I'm not sure if changing this class is necessary.

Capture d’écran 2022-11-10 à 12 11 02

**Review by Brieuc-brd — CHANGES_REQUESTED** [review]
Hey @xlu-odoo 👋 
Some minor things, thanks !

**Review by Brieuc-brd — APPROVED** [review]
LGTM 👍 
Hello @qsm-odoo this PR is ready for a technical review, could you take a look at it ? 🙂

**Review by Williambraecky — COMMENTED** [review]
Hello :)
Have those changes been validated with @pierre-pde ?
Cheers

**Review by Williambraecky — APPROVED** [review]
Thanks :)
robodoo r+

**Pull request closed without merging**